runatlantis.io/docs/stats.md

```diff
@@ -9,3 +9,45 @@ Currently Statsd and Prometheus is supported. See configuration below for detail
 ## Configuration
 
 Metrics are configured through the [Server Side Config](server-side-repo-config.html#metrics).
+
+## Available Metrics/Stats
+
+There are plenty of metrics which are exposed by atlantis service. Let's say you have exposed metrics from a endpoint `/metrics` from your [Server Side Config](server-side-repo-config.html#metrics)
+```
+metrics:
+  prometheus:
+    endpoint: "/metrics"
+```
+then on making curl request to your atlantis server on path `/metrics` will list down all the metrics exposed from atlantis service.
+```
+curl localhost:4141/metrics
+# HELP atlantis_cmd_autoplan_builder_execution_error atlantis_cmd_autoplan_builder_execution_error counter
+# TYPE atlantis_cmd_autoplan_builder_execution_error counter
+atlantis_cmd_autoplan_builder_execution_error 0
+# HELP atlantis_cmd_autoplan_builder_execution_success atlantis_cmd_autoplan_builder_execution_success counter
+# TYPE atlantis_cmd_autoplan_builder_execution_success counter
+atlantis_cmd_autoplan_builder_execution_success 10
+# HELP atlantis_cmd_autoplan_builder_execution_time atlantis_cmd_autoplan_builder_execution_time summary
+# TYPE atlantis_cmd_autoplan_builder_execution_time summary
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.5"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.75"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.95"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.99"} NaN
+atlantis_cmd_autoplan_builder_execution_time{quantile="0.999"} NaN
+atlantis_cmd_autoplan_builder_execution_time_sum 11.42403017
+atlantis_cmd_autoplan_builder_execution_time_count 10
+.....
```

Comment:
Might as well list all of them, no?

Reply:
The whole output is 300+ lines, which will take up more space on documentation page and with every new version release this metric set will need to updated accordingly as there may be case if some metrics are added/deprecated, so it's better to just give a brief idea of how they will look like and rest can be explored. Let me know if there any issues with this approach.

Reply:
Good point. Could you mention this on the page to state that that is why the output is trimmed?

Reply:
mentioned at https://github.com/runatlantis/atlantis/pull/3171/files#diff-ed3d3eb5ab28409011a42ca500407d05273e7ecf5c5a4143bba27b5a95220972R43